doc/whats-new.rst

```diff
@@ -181,6 +181,9 @@ Documentation
 
 Enhancements
 ~~~~~~~~~~~~
+- added backend for many Atmospheric data formats ranging including GEOS-Chem,
+  CAMx, NOAA arlpacked bit and many others.
+- reduce methods such as :py:func:`DataArray.sum()` now accepts ``dtype``
```

Comment:
please delete

Reply:
Not sure what you're referring to since I am also editing and moving the first two lines? The third line was not mine.